{"name": "test_search_api_invalid_auth", "status": "passed", "steps": [{"name": "Get countries api hit with invalid auth", "status": "passed", "start": 1707021418339, "stop": 1707021418339}], "start": 1707021417279, "stop": 1707021418341, "uuid": "12f7b642-9c93-4888-a4b9-2b0e63e470b2", "historyId": "4e84a5577876079021e45447db705ef0", "testCaseId": "4e84a5577876079021e45447db705ef0", "fullName": "step_defs.test_countries#test_search_api_invalid_auth", "labels": [{"name": "parentSuite", "value": "step_defs"}, {"name": "suite", "value": "test_countries"}, {"name": "host", "value": "DESKTOP-7LBLCDO"}, {"name": "thread", "value": "10024-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "step_defs.test_countries"}]}